airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/spec.json

```diff
@@ -14,12 +14,6 @@
         "description": "Date in the format 2020-09-17. Any data before this date will not be replicated.",
         "examples": ["2021-05-17"]
       },
-      "access_token": {
```

Comment:
can I confirm we are getting rid of the access token?

Reply:
No, both auth methods can be available. User will be able to use the new oAuth2 logic or use own access token directly
![Screenshot 2021-11-18 at 00 52 58](https://user-images.githubusercontent.com/11213273/142295323-2906c827-e830-4ad7-97fd-68e970e14d45.png)
.